src/frontend.rs

```diff
@@ -314,22 +339,17 @@ impl MainPodBuilder {
             // main_pods: &self.input_main_pods,
             statements: &self.statements,
             operations: &self.operations,
+            public_statements: &self.public_statements,
         };
-        let (statements, operations) = compiler.compile(inputs)?;
+        let (statements, operations, public_statements) = compiler.compile(inputs)?;
 
+        // TODO: Add API to specify public/private statement
```

Comment:
@arnaucube this is already supported with the `fn pub_op` and `fn reveal` methods, I forgot to remove this comment.

Reply:
thanks, removed the comment at https://github.com/0xPARC/pod2/pull/35